src/kakarot/instructions/block_information.cairo

```diff
@@ -122,6 +124,14 @@ namespace BlockInformation {
         Stack.push_uint128(evm.message.env.base_fee);
         jmp end;
 
+        invalid_opcode_0x49:
+        jmp end;
```

Comment:
this will just create a cairovm error down the line.

```
let stack = cast([fp - 6], model.Stack*);
tempvar stack = stack;
```

would make this opcode transparent, but we want to revert the EVM instead right?

Reply:
Basically this would never be reached because in the global opcodes jmp table 0x49 just jumps to invalid opcode.

But either way: 0x49 is blobhash so this will disappear after a rebase with the other pr